cmd/cosign/cli/sign/sign.go

```diff
@@ -591,3 +617,33 @@ func (c *SignerVerifier) Bytes(ctx context.Context) ([]byte, error) {
 	}
 	return pemBytes, nil
 }
+
+func fetchSignedPayload(sig oci.Signature) (*cosign.SignedPayload, error) {
```

Comment:
Can this return a `LocalSignedPayload` instead to be in sync with sign-blob? It should be loadable with https://github.com/sigstore/cosign/blob/main/pkg/cosign/fetch.go#L199.